examples/meta/generator/translate.py

```diff
@@ -412,6 +412,12 @@ def getIncludePathForClass(self, type_):
                                      uniqueCandidates[0]))
             return uniqueCandidates[0]
 
+        # hack since ctags cannot find include paths for preprocessor
+        # macros generated names that use the concat (##) operator
```

Comment:
something is wrong with your git now, as this is from another patch

Reply:
Yes it's strange, these all not my code. Will try to find a way how to solve .